gk/fib.c

```diff
@@ -1825,7 +1832,8 @@ list_ipv4_fib_entries(lua_State *l, struct gk_lpm *ltbl)
 		dentry.addr.ip.v4.s_addr = htonl(re4->ip);
 		dentry.prefix_len = state.depth;
 		fib = &ltbl->fib_tbl[re4->next_hop];
-		ret = fillup_gk_fib_dump_entry(&dentry, fib);
+		ret = fillup_gk_fib_dump_entry(&dentry, fib,
+			memblock_from_stack(stack_mb));
```

Comment:
When compiling the code, there is an error: ``error: implicit declaration of function ‘memblock_from_stack’; did you mean ‘memblock_free_block’? [-Werror=implicit-function-declaration]``. Indeed, I didn't find this function.

Reply:
You need to rebase with the repository to see `memblock_from_stack().`

Reply:
I see. There was another latest patch adding exactly this.